### DISCUSSION TITLE
How do I add different user types with authentication?

### DISCUSSION THREAD

**Discussion by westdabestdb** [strapi/strapi#7211] [Legacy Topics]
My application is supposed to have Facilities and Patients, and each of those should be able to login with their credentials. Is there a way to configure Strapi to handle this?

**Comment by lauriejim** [comment]
Hello! You can customize a bit your register function to manage both roles.
With the customization concept you will be able to customize the plugin function - https://strapi.io/documentation/v3.x/concepts/customization.html
Here is the thing you have to customize - https://github.com/strapi/strapi/blob/master/packages/strapi-plugin-users-permissions/controllers/Auth.js#L419
